src/lib.rs

```diff
@@ -59,13 +59,17 @@ mod comparator;
 mod db;
 mod db_options;
 pub mod merge_operator;
+mod optimistictransactiondb;
```

Comment:
I'd propose to split this name with underscore. `optimistic_transaction_db` or even `optimistic_tx_db`. What do you think ?

Reply:
good